auto_nag/scripts/vacant_triage_owner.py

```diff
@@ -110,8 +113,44 @@ def identify_vacant_components(self):
 
         return vacant_components
 
-    def get_email_data(self, date, bug_ids):
-        return self.identify_vacant_components()
+    def populate_num_untriaged_bugs(self, components: List[dict]) -> None:
+        """Add the number of untriaged bugs and the URL to its search query to
+        the provided components.
+
+        The method will mutate the provided dictionaries to add the results.
+        """
+
+        def handler(bug, data):
+            data["untriaged_bugs"] += 1
+
+        queries = [None] * len(components)
+        for i, component in enumerate(components):
+            params = {
+                "resolution": "---",
+                "severity": "--",
+                "product": component["product"],
+                "component": component["component"],
+            }
```

Comment:
We should restrict to defects, because they are the only type of bugs that require a severity to be set.
In the future we'll switch to the `triaged` keywords and then we will be able to remove the filter on type.

Reply:
Done in 27f952083449ef2b88520012100a70425697cb87